bids-validator/src/schema/context.ts

```diff
@@ -170,13 +170,21 @@ export class BIDSContext implements Context {
       return
     }
     const sidecars = walkBack(this.file)
```

Comment:
I like to keep try/catches very small what about this instead:

```suggestion
    try {
      const sidecars = [...walkBack(this.file)]
    } catch (error) {
      if (error.code === 'MULTIPLE_INHERITABLE_FILES') {
        this.dataset.issues.add(error)
      } else {
        throw error
      }
    }
```